### DISCUSSION TITLE
Option to only show x entries

### DISCUSSION THREAD

**Discussion by Bense89** [totaldebug/atomic-calendar-revive#1515] [Support]
I use the calendar as an integration into a dashboard. Each module has a fixed size, except the calendar, as the number of entries varies. Is there an option where I can only see X entries so that the size is always the same? 

The only option I see is reduced to days. But this also causes the "problem" that I see, as an example, 30 or 34 entries, which changes the size of the module.

**Comment by marksie1988** [comment]
`maxEventCount` may be what you need

**Comment by Bense89** [comment]
Oh man, I didn't find it. Thanks!

**Comment by Bense89** [comment]
maxEventCount
